go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/elasticsearch-proxy
 
-go 1.18
+go 1.19
```

Comment:
No changes in `go.sum` with `go mod tidy` after this?

Reply:
nope